Tutorial updates

jomey opened the pull request:
Since I poked the hornets nest to tried to build the website nearly two years later, I also went ahead and fixed the existing typos. 

Main changes:
* Update a git link for tutorial data that has been moved
* Fix a couple of typos raised by QA/QC

jomey committed:
Tutorials - UAVSAR - Update git url

The repository has been moved to the SnowEx organization.

jomey committed:
Tutorials - Fix typos raised by QA/QC

jomey commented on book/tutorials/uavsar/2_types_of_uavsar_images.ipynb:
```diff
@@ -106,11 +106,11 @@
     "os.chdir('/tmp/')\n",
-    "if not os.path.exists('/tmp/uavsar-tutorial-data'):\n",
-    "    !git clone --quiet https://github.com/SnowEx/uavsar-tutorial-data.git\n",
+    "if not os.path.exists('/tmp/uavsar-tutorial-data-2022'):\n",
```
This was the main reason having this PR

JessicaS11 reviewed (changes requested):
The build can't find a needed data file to render outputs...

jomey commented:
> The build can't find a needed data file to render outputs...

Yup. Just found another (hopefully) last path that was outdated.

jomey committed:
Tutorials - UAVSAR - Update plot to work in 2024

Unfortunately the call to add terrain lables to a plot using
contextily does not render. Only a small stylistic change and the main
content of the tutorial is still preserved.

jomey commented on book/tutorials/uavsar/3_interferometric_swe_inversion.ipynb:
```diff
@@ -174,7 +174,8 @@
     "# Add background map\n",
-    "cx.add_basemap(ax, crs=img.rio.crs, source = cx.providers.Stamen.TerrainLabels)\n",
+    "# Unfortunately, the below line does not render as of 20240301\n",
+    "# cx.add_basemap(ax, crs=img.rio.crs, source = cx.providers.Stamen.TerrainLabels)\n",
```
Sad to see this not working with today's run anymore. Luckily it is not breaking essential parts of the tutorial.

scottyhq replied:
Yes, sadly since 10/2023 Stamen-provided map tiles require an API key (https://xyzservices.readthedocs.io/en/stable/changelog.html#xyzservices-2023-10-1-october-26-2023)

jomey replied:
Thanks for sharing and good to know

jomey committed:
Book - Remove event quick links

jomey commented:
I am accepting the failure with OS X, given that we can build the website and we have a successful Linux build too.
